src/Components/SmartComponents/Reports/DownloadCVEsReport.js

```diff
@@ -69,6 +69,11 @@ const DownloadCVEsReport = ({
             ...cve,
             attributes: {
                 ...cve.attributes,
+                status: STATUS_OPTIONS.find(item =>
+                    item.value === cve.attributes.status_id.toString()).label,
+                business_risk: BUSINESS_RISK_OPTIONS.find(item =>
+                    item.value === cve.attributes.business_risk_id.toString()).label,
+                impact: impactList[cve.attributes.impact].title,
```

Comment:
it works for the reports page but it breaks for the landing page. I suspect it's because in some cases CVE can have `notSet` impact and the `impactList` doesn't have such a value.

Reply:
```
Uncaught (in promise) TypeError: _Helpers_constants__WEBPACK_IMPORTED_MODULE_16__.impactList[cve.attributes.impact] is undefined
```

Reply:
Good catch, fixed